content/en/continuous_integration/troubleshooting.md

```diff
@@ -74,6 +74,26 @@ If you can see test results data in the **Test Runs** tab, but not the **Tests**
 
 4. If no CI provider environment variables are found, tests results are sent with no Git metadata.
 
+### The tests wall time is empty
+
+If you cannot see the tests wall time it is likely that CI provider metadata is missing. To confirm this is the case, open a test execution in the [Test Runs][3] section, and check if `ci.pipeline.id`, `ci.pipeline.name`, `ci.pipeline.number` or `ci.job.url` tags are missing. If these tags are not populated then nothing shows in the wall time column.
+
+1. Tracers use the environment variables set by the CI provider to collect this information. See [Running tests inside a container][6] for a list of environment variables that the tracer attempts to read for each supported CI provider. Make sure that this environment variables have the expected values set.
+2. Check you are running your tests in a supported CI provider, for a list of supported CI providers see [Running tests inside a container][6]. Only these CI providers can extract the information to enrich the test metadata with CI information.
```

Comment:
```suggestion
2. Check that you are running your tests in a supported CI provider. For a list of supported CI providers, see [Running tests inside a container][6] documentation. Only these CI providers can extract the information to enrich the test metadata with CI information.
```

Reply:
Similar to line 81 above, this one also needs the word "the" in front of "[Running tests inside...]"